README.md

```diff
@@ -22,15 +22,18 @@
 Windows is not an explicitly supported environment, although where possible Palisade has been made compatible.  
 For Windows developer environments, we recommend setting up [WSL](https://docs.microsoft.com/en-us/windows/wsl/).
 
+## Getting started
+For an overview of Palisade, start with root documentation [Palisade README](https://github.com/gchq/Palisade#readme).
```

Comment:
Would something like this read better?
```suggestion
This is not the starting point for understanding Palisade.
This should be done from within the main [Palisade repo](https://github.com/gchq/Palisade)
```

Reply:
I went for another tack and expanded on what is at the other end of the link "For an overview of Palisade, start with the Palisade introduction and the accompanying guides: QuickStart Guide; and Developer Guide:"